packages/discord.js/src/structures/Role.js

```diff
@@ -177,7 +177,7 @@ class Role extends Base {
    */
   get editable() {
     if (this.managed) return false;
-    const clientMember = this.guild.members.resolve(this.client.user);
+    const clientMember = this.guild.members.me;
```

Comment:
This should be reverted, it's unrelated to the PR.

Reply:
done